### PR TITLE
ceph: kubectl edit should be kubectl patch

### PR DESCRIPTION
**Pull request by oneoneonepig**
**Description of your changes:**

As mentioned in title, the kubectl edit command should be kubectl patch

Signed-off-by: Jeffrey Chu <peihuachu1112@gmail.com>

**Checklist:**

- [X] Documentation has been updated, if necessary.
- [X] Add the flag for skipping the CI if this PR does not require a build. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for more details.

[skip ci]

### CODE REVIEW

File: Documentation/disaster-recovery.md [hunk 1]
```diff
@@ -196,19 +196,21 @@ The following `kubectl patch` command is an easy way to do that. In the end it p
...
-kubectl -n rook-ceph edit secret rook-ceph-config -p '{"stringData": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "'"${good_mon_id}"'"}}'
+kubectl -n rook-ceph patch secret rook-ceph-config -p '{"stringData": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "'"${good_mon_id}"'"}}'
 ```
 
 > **NOTE**: If you are using `hostNetwork: true`, you need to replace the `mon_host` var with the node IP the mon is pinned to (`nodeSelector`). This is because there is no `rook-ceph-mon-*` service created in that "mode".
 
 ### Restart the mon
 
-You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl replace` on the backup of the mon deployment yaml:
+You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl delete` to restart the pod. :
```
Comment (leseb) [inline Documentation/disaster-recovery.md]:
```suggestion
You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl delete` to restart the pod:
```

Comment (oneoneonepig) [reply Documentation/disaster-recovery.md]: Fixed

Comment (leseb) [reply Documentation/disaster-recovery.md]: You don't need a commit for this since you introduced that typo. Just amend the commit that introduced it.

**Comment by travisn** [comment]
@oneoneonepig Could you just fix the commit messages according to the bot feedback? For example, your commit message could be:
```
ceph: kubectl edit should be kubectl patch

<short paragraph describing the change>
```

**Comment by oneoneonepig** [comment]
@travisn 

I have a question, since I'm not that familiar to all those commits/PRs, I happened to committed two different fixes in a same branch, one is "changing kubectl edit to kubectl patch", the other one is "adding --force to kubectl replace".

Should I reset the current branch and keep the first fix, and open another branch to fix the second issue; or should I keep both in the same PR?

I'll fix the content of the commit after I'm certain what I am doing, thanks.

**Comment by travisn** [comment]
@oneoneonepig That's fine to keep both commits in this PR, no need to split them up into a separate PR. The commit messages just need to be updated to satisfy the bot.

**Pull request merged**